### PR TITLE
kiwix: Add version 2.3.1

### LINKED ISSUE #10156: [Request]: Kiwix
### Prerequisites

- [X] I have searched all issues/PRs to ensure it has not already been reported or fixed.

### Criteria

- [X] Reasonably well-known and widely used (e.g. if it's a GitHub project, it should have at least 100 stars and/or 50 forks)
- [X] English interface (or at least English documentation)
- [X] Latest stable version
- [X] Full version (i.e. not a trial version)
- [X] Fairly standard install (e.g. uses a version-specific download URL, no elaborate pre/post install scripts)

### Name

Kiwix

### Description

Store any website on your mobile phone or computer, easily.  Offline Wikipedia, Project Gutenberg reader

### Homepage

https://www.kiwix.org/en/

### Download Link(s)

https://www.kiwix.org/en/download/

### Some Indication of Popularity/Repute

Official desktop app repo:   https://github.com/kiwix/kiwix-desktop

425 stars and 67 forks

### PR DESCRIPTION
**Pull request by Zliced13**
Closes #10156

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/kiwix.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
+    "version": "2.3.1",
+    "homepage": "https://www.kiwix.org/en/",
+    "description": "Store any website on your mobile phone or computer, easily.",
+    "license": "GPL-3.0",
```
Comment (HUMORCE) [inline bucket/kiwix.json:5]:
```suggestion
    "license": "GPL-3.0-only",

```

File: bucket/kiwix.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    ],
+    "checkver": {
+        "script": [
+            "$userAgent = 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/103.0.0.0 Safari/537.36'",
+            "$returnFile = Invoke-WebRequest 'https://download.kiwix.org/release/kiwix-desktop/kiwix-desktop_windows_x64.zip' -UserAgent $userAgent",
+            "$returnFile = ($returnFile.BaseResponse.RequestMessage.RequestUri.AbsoluteUri) -replace 'https://.*/(kiwix-desktop_windows_x64_[\\d.]+\\.zip)', '$1'",
+            "Write-Output $returnFile"
+        ],
+        "regex": "kiwix-desktop_windows_x64_([\\d.]+)\\.zip"
+    },
```
Comment (HUMORCE) [inline bucket/kiwix.json:28]: https://github.com/kiwix/kiwix-desktop

**Review by HUMORCE — CHANGES_REQUESTED** [review]
`license` and `checkver`

File: bucket/kiwix.json (hunk 2 above)
Comment (Zliced13) [reply bucket/kiwix.json:28]: That `checkver` is not going to work @HUMORCE.

Comment (Zliced13) [reply bucket/kiwix.json:28]: There is a reason why I have this `checkver` script to get the version. And, if you tested it for yourself, you would know why.

https://www.dropbox.com/s/oo23xzlnt1k8n0t/kiwix_checkver.ps1?dl=0

Comment (HUMORCE) [reply bucket/kiwix.json:28]: https://download.kiwix.org/release/kiwix-desktop/feed.xml

**Comment by Zliced13** [comment]
@HUMORCE I changed the checkver script, I'm not familiar with the `xpath` property, so I used my own PowerShell script instead.

File: bucket/kiwix.json [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        ]
+    ],
+    "checkver": {
+        "script": [
+            "$userAgent = 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/103.0.0.0 Safari/537.36'",
+            "((Invoke-RestMethod 'https://download.kiwix.org/release/kiwix-desktop/feed.xml' -UserAgent $userAgent).title)[0] -match '(kiwix-desktop_windows_x64_[\\d.]+\\.zip)' | Out-Null",
+            "$returnFile = $Matches[0]",
+            "Write-Output $returnFile"
+        ],
+        "regex": "kiwix-desktop_windows_x64_([\\d.]+)\\.zip"
```
Comment (HUMORCE) [inline bucket/kiwix.json:27]:
```suggestion
        "url": "https://download.kiwix.org/release/kiwix-desktop/feed.xml",
        "regex": "kiwix-desktop_windows_x64_([\\d.]+)\\.zip"
```

**Review by HUMORCE — APPROVED** [review]

**Pull request merged**